themes/bootstrap3/templates/Recommend/EDSResultsDeferred.phtml

```diff
@@ -1,9 +1 @@
-<?php
-    // Set up Javascript for use below:
-    $loadJs = 'var url = VuFind.path + "/AJAX/Recommend?' . $this->recommend->getUrlParams() . '";'
-        . "VuFind.loadHtml(\$('#edsResultsDeferredRecommend'), url);";
-?>
-<div id="edsResultsDeferredRecommend">
-    <p><?=$this->icon('spinner') ?> <?=$this->transEsc('loading_ellipsis')?></p>
-    <?=$this->inlineScript(\Laminas\View\Helper\HeadScript::SCRIPT, $loadJs, 'SET')?>
-</div>
+<?=$this->deferredRecommend('edsResultsDeferredRecommend');
```

Comment:
As I start to build on the other template files, I see even this variable name is irrelevant -- could be a random generated string as long as it's unique -- since it's just used two places within the Deferred.phtml template.  But I think it's a valuable hook anyway for CSS.

Reply:
Yeah, I think it's okay to leave it for that reason.